.github/workflows/build.yaml

```diff
@@ -0,0 +1,59 @@
+name: "build"
+on:
+  push:
+    branches:
+    - "!dependabot/*"
+    - "*"
+  pull_request:
+    branches:
+    - "*"
+jobs:
+  lint: 
+    name: "Lint"
+    runs-on: "ubuntu-latest"
+    steps:
+    - uses: "actions/checkout@v2"
+    - uses: "actions/setup-go@v2"
+      with:
+        go-version: ^1.16
+    - name: "go fmt"
+      run: |
+        GOFMT_OUTPUT="$(gofmt -l `find ./ -name '*.go' | grep -v vendor` 2>&1)"
```

Comment:
I avoided the premade github actions because none of them use the same version of go that you install with setup-go and this is all very simple stuff.